cmd/web6/web6.go

```diff
@@ -23,10 +22,10 @@ func main() {
 
 	// We dont use NewFromFlag() because it doesnt allow adding custom flags.  We have to manage the gosrv config Struct ourselves in config.go
 	/*
-	s, err := gosrv.NewFromFlag()
-	if err != nil {
-		panic("Cannot create web server: " + err.Error() + "\n")
-	}*/
+		s, err := gosrv.NewFromFlag()
```

Comment:
Did you test this with the custom options?  It doesnt respect my flag options when it starts in this way previously.

I havent tested this yet, just not sure if you saw the comment above and knew the history with the flag issue?

Reply:
go imports fixed the spacing, nothing else changed here.